judge/models/runtime.py

```diff
@@ -123,6 +128,12 @@ class Judge(models.Model):
     def __unicode__(self):
         return self.name
 
+    def disconnect(self, force=False):
+        if self.online:
```

Comment:
A judge might be online in a phantom state. This is known to occur when a judge disconnects due to network failure, and then the judge reinstates the connection. However, the bridge realizes the judge disconnected afterwards, and the judge becomes marked as offline, despite having another connection. We need to significantly rework the bridge to fix this issue, but in the meantime... we probably shouldn't check `self.online`.